app/src/main/java/com/infomaniak/swisstransfer/ui/images/illus/onboarding/StorageBoxPileLight.kt

```diff
@@ -0,0 +1,633 @@
+package com.infomaniak.swisstransfer.ui.images.illus.onboarding
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.PathFillType.Companion.EvenOdd
+import androidx.compose.ui.graphics.PathFillType.Companion.NonZero
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.graphics.StrokeCap.Companion.Butt
+import androidx.compose.ui.graphics.StrokeJoin.Companion.Miter
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.graphics.vector.ImageVector.Builder
+import androidx.compose.ui.graphics.vector.group
+import androidx.compose.ui.graphics.vector.path
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.infomaniak.swisstransfer.ui.images.AppImages
+import com.infomaniak.swisstransfer.ui.images.AppImages.AppIllus
+
+val AppIllus.StorageBoxPileLight: ImageVector
+    get() {
+        if (_storageBoxPileLight != null) {
+            return _storageBoxPileLight!!
+        }
+        _storageBoxPileLight = Builder(
+            name = "StorageBoxPileLight",
+            defaultWidth = 290.0.dp,
+            defaultHeight = 258.0.dp,
+            viewportWidth = 290.0f,
+            viewportHeight = 258.0f
+        ).apply {
+            path(
+                fill = SolidColor(Color(0xFFA98116)),
+                stroke = null,
```

Comment:
Stroke parameters can be removed for a lot of paths in this file.

Reply:
It's probably the same in the following illustrations, I'll let you check them.

Reply:
Yeah I removed all null stroke I found in the illustrations